python/tests/stress/arcticdb/version_store/test_large_df.py

```diff
@@ -59,6 +62,8 @@ def test_write_and_update_large_df_in_chunks(lmdb_version_store_very_big_map):
     lib.update(symbol, expected_head)
     assert_frame_equal(lib.head(symbol).data, expected_head)
 
+
+@pytest.mark.skip(reason="Uses too much storage and fails on GitHub runners")
```

Comment:
But it presumably does work at the moment? Better to adjust the parameters if needed than skip completely